code/style.css

```diff
@@ -1,4 +1,309 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/** HEADER **/
+
+header {
+  padding: 10px 20px;
+  background: #FFFFFF;
+  color: #121212;
+  font-family: Helvetica;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  header {
+    justify-content: space-evenly;
+  }
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  /*to spread the lines, since they are stacked*/
+  justify-content: space-between;
+  height: 12px;
+  width: 18px;
+  margin: 0px, 0px, 0px, -80px;
```

Comment:
1. There shouldn't be any comma between values.
2. Moreover, if you want to add value `0px`, you can just omit `px` and type only `0`. Final proper result should look like this : `margin: 0 0 0 -80px`.
3. Adding margins with value lower than zero is not the best idea, wasn't there better solution for that? Maybe you could add margin equal to `+80px` somewhere else?